app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.kt

```diff
@@ -2114,54 +2119,54 @@ class NearbyParentFragment : CommonsDaggerSupportFragment(),
         return drawableCache!![key]
     }
 
-    fun convertToMarker(place: Place, isBookMarked: Boolean): Marker {
```

Comment:
what did you change here? Please do not change formatting, it makes changes less understandable.